bin/create-tokens/main.go

```diff
@@ -93,18 +92,20 @@ func main() {
 			log.Fatalf("%s is not a valid uuidv4\n", *promotionID)
 		}
 	}
-
-	if len(grantSignatorPrivateKeyHex) == 0 {
-		log.Fatalln("Must pass grant signing key via env var GRANT_SIGNATOR_PRIVATE_KEY")
+	if err != nil {
```

Comment:
is there an unchecked error above? I think we already checked all instances as they happened - in which case we can remove this extra check